src/test/java/org/jenkinsci/plugins/matrixauth/AuthorizationContainerDescriptorTest.java

```diff
@@ -37,14 +34,8 @@ public void testImpliedNotes() {
             Assert.assertFalse(description.contains(Messages.GlobalMatrixAuthorizationStrategy_PermissionImpliedBy(Jenkins.PERMISSIONS.title.toString(), Jenkins.ADMINISTER.name)));
         }
 
-        { // Item.CANCEL is implied by Item.BUILD until 2.119. From 2.120 is implied by Permission.UPDATE, so the description changes.
```

Comment:
You have more context about if it's worth to maintain the test for Item.CANCEL, so I'm happy to remove it.

Reply:
Simply chosen as an example of its category of being implied by a non-Overall/Administer permission.